sqlx-cli/README.md

```diff
@@ -20,8 +20,6 @@ $ cargo install --version=0.2.0 sqlx-cli --no-default-features --features postgr
 All commands require that a database url is provided. This can be done either with the `--database-url` command line option or by setting `DATABASE_URL`, either in the environment or in a `.env` file
 in the current working directory.
 
-`database` and `migrate` subcommands support only Postgres; MySQL and SQLite are TODO.
-
```

Comment:
Technically, MSSQL isn't supported _at all_ right now so that should probably be mentioned here. Otherwise, looks fine to me.